.travis.yml

```diff
@@ -72,21 +72,25 @@ branches:
     - master
     - /^\d+\.\d+\.\d+.*$/
 
-
 notifications:
   slack: typekit:TRAVIS_SLACK_TOKEN
   email:
     on_success: change
     on_failure: always
 
+before_deploy:
+  - ./.travis/before_deploy.sh
+
 deploy:
-  provider: pypi
-  server: https://upload.pypi.org/legacy/
-  user: afdko-travis
-  password:
-    secure: fkjaRurDm3ccb9IUPqDNd5VX1PkZnPrraedHigbpRmc11e0xn+AwWEBuI9EhFLlIl3omzxmajw4UXQb2qw0pIyLPUGnASZvsHWFE9mUz6v2XHYSI1RLG+xpyd0F7ZGExAW6U5fca1uTUTIo68/CJCMk0NSrcVGi6WkaFOXsQ+45UX7uSfCvPsA80w1DnA3tgrrbaLPaz2y9IZmdmMU8DQzE5IU8ODtXS1exQV1YUhlapRvvOkmhv61edVlTeqzd9dCQPWHxegqs6hkiUVX01H0wYNQkaomY6K1O8AmOmDvacXRQILkB/H5luQNGtOhRMGIwr65VIjLl2xNVgEzVAT3rYQ31oz794AkpjDGvzccNgUT9BjwVjxbBrYqpltfYs6q8HlQ8etl1GMjMXrHoZLdNlemKsyAghpNOrQD9pTHYM8OU7m6tCxSDzLnMwnGpjHohsfDg6WWfoBybyAK2RwJ0/Okv/lgdPdupaPOmsJb8h0ymuQB7StAoWYAql2VO+JrFOLlBar+n/2LtjVvW6KISpDSE7qrJEtEfO0ElU8Pc/T+cE5qBlBxa3wh87nqBPee/oOcJNuNmHCk7YzWCXShj+fiDjPQYy6WLHZ1/e8I1AV3DEWlqOAnGja46FARRHLZ8GNLhZ8DAiRsgV4rd9NCl3v7UJWp036Tr7KAZP2nc=
-  distributions: bdist_wheel
-  on:
-    tags: true
-    repo: adobe-type-tools/afdko
-    condition: "$BUILD_DIST == true"
+  # deploy to PyPI on tags
+  - provider: pypi
+    server: https://upload.pypi.org/legacy/
+    user: afdko-travis
+    password:
+      secure: fkjaRurDm3ccb9IUPqDNd5VX1PkZnPrraedHigbpRmc11e0xn+AwWEBuI9EhFLlIl3omzxmajw4UXQb2qw0pIyLPUGnASZvsHWFE9mUz6v2XHYSI1RLG+xpyd0F7ZGExAW6U5fca1uTUTIo68/CJCMk0NSrcVGi6WkaFOXsQ+45UX7uSfCvPsA80w1DnA3tgrrbaLPaz2y9IZmdmMU8DQzE5IU8ODtXS1exQV1YUhlapRvvOkmhv61edVlTeqzd9dCQPWHxegqs6hkiUVX01H0wYNQkaomY6K1O8AmOmDvacXRQILkB/H5luQNGtOhRMGIwr65VIjLl2xNVgEzVAT3rYQ31oz794AkpjDGvzccNgUT9BjwVjxbBrYqpltfYs6q8HlQ8etl1GMjMXrHoZLdNlemKsyAghpNOrQD9pTHYM8OU7m6tCxSDzLnMwnGpjHohsfDg6WWfoBybyAK2RwJ0/Okv/lgdPdupaPOmsJb8h0ymuQB7StAoWYAql2VO+JrFOLlBar+n/2LtjVvW6KISpDSE7qrJEtEfO0ElU8Pc/T+cE5qBlBxa3wh87nqBPee/oOcJNuNmHCk7YzWCXShj+fiDjPQYy6WLHZ1/e8I1AV3DEWlqOAnGja46FARRHLZ8GNLhZ8DAiRsgV4rd9NCl3v7UJWp036Tr7KAZP2nc=
+    skip_cleanup: true
+    distributions: pass
```

Comment:
yeah, that "pass" thing is simply a way to trick the Travis dpl tool to run `python setup.py pass` and in fact _not_ do anything. The default is that it would do `python setup.py sdist bdist_wheel`. But if you have already built your distribution packages (e.g. for C extension modules we want to compile them inside a manylinyux1 docker container for greater portability), we don't want Travis to attempt to rebuilt them. If we `skip_cleanup` and place the sdist and wheel packages inside the default `dist/` folder, then the Travis dpl tool will find them and upload those instead of rebuilding them again as part of the deploy stage.

Lately, instead of defining a custom setuptools "pass" command that does nothing, I've been passing the built-in `check` command to the deploy `distributions:` to achieve the same effect. Doing `setup.py check` does basic metadata checks (may return non-zero if something is not ok), but does not build anything.

Reply:
I must have done something wrong because AFAICT Travis is trying to build them.